tempesta_fw/cache.c

```diff
@@ -41,6 +41,11 @@
 #warning "Please set CONFIG_NODES_SHIFT to less than 16"
 #endif
 
+/* Flags stored in a Cache Entry. */
+#define TFW_CE_MUST_REVAL	0x0001		/* MUST revalidate if stale. */
+#define TFW_CE_NO_CACHE		0x0002		/* ALWAYS revalidate. */
```

Comment:
If a response has the `no-cache` CC directive, then according to the RFC 7234 5.2.2.2 that means that the response may be stored in cache. However, it must not be returned to a client without a revalidation. That is especially beneficial for voluminous responses, as a revalidation sequence it much smaller and therefore faster. Are we not going to implement this feature and just bypass the cache?